app/controllers/provider_interface/offers_controller.rb

```diff
@@ -101,6 +100,7 @@ def available_course_options(course_id, study_mode)
     end
 
     def offer_context_params(course_option, conditions = MakeAnOffer::STANDARD_CONDITIONS, decision = :default)
+      further_conditions_params = (conditions - MakeAnOffer::STANDARD_CONDITIONS).map.with_index.to_h { |c, index| [index.to_s, { 'text' => c }] }
```

Comment:
I'm going to move this into a `build_from_application_choice` method inside the wizard, since the controller will be cleaner if it just passes params to the wizard